### PR TITLE
fix for issue #30 brightness

### PR DESCRIPTION
**Pull request by max9901**

### CODE REVIEW

File: src/hal/display.cpp [hunk 1]
```diff
@@ -90,11 +90,21 @@ void OswHal::increaseBrightness(uint8_t v) {
   } else {
     _brightness += v;
   }
+  #ifdef ESP32
```
Comment (uvwxy) [inline src/hal/display.cpp:93]: could you change this to setBrightness(_brightness)

would be less code duplication that way. thank you :)

File: src/hal/display.cpp [hunk 2]
```diff
@@ -90,11 +90,21 @@ void OswHal::increaseBrightness(uint8_t v) {
...
+    digitalWrite(TFT_LED, brightness);
+  #endif
 };
 void OswHal::decreaseBrightness(uint8_t v) {
   if (_brightness < v) {
     _brightness = 0;
   } else {
     _brightness -= v;
   }
+  #ifdef ESP32
```
Comment (uvwxy) [inline src/hal/display.cpp:105]: see above 👆

**Review by uvwxy — CHANGES_REQUESTED** [review]
If you change the setting of the brightness to use the existing setter function, I can merge this

File: src/hal/display.cpp (hunk 1 above)
Comment (max9901) [reply src/hal/display.cpp:93]: Yeah but its 1 more memory operation. so slower.

Comment (max9901) [reply src/hal/display.cpp:93]: But yeah if this is "cleaner" in your opinion i can change it ofcourse!

Comment (uvwxy) [reply src/hal/display.cpp:93]: I understand you.

My perspective was: fewer places to change stuff if we eventually use a different display.
Also 1 more memory operation versus 240Mhz on a function that currently get's called once when the watch wakes up and once when it goes back to sleep. Even gradually dimming at 100fps would still leave tons of overhead of performance...

**Review by uvwxy — APPROVED** [review]

**Pull request merged**